src/engine/sync/basesyncablelistener.h

```diff
@@ -1,71 +0,0 @@
-#pragma once
-
-#include "engine/sync/syncable.h"
-#include "preferences/usersettings.h"
-
-class InternalClock;
-class EngineChannel;
-
-/// BaseSyncableListener is a SyncableListener used by EngineSync.
-/// It provides some foundational functionality for distributing
-/// various sync updates.
-// TODO(owilliams): This class is only inherited by EngineSync. It should
-// be merged into that class.
```

Comment:
I was confused why the functionality was split across these classes, then I found this comment.